sdk/azcore/runtime/policy_retry.go

```diff
@@ -102,7 +102,8 @@ func (p *retryPolicy) Do(req *policy.Request) (resp *http.Response, err error) {
 	try := int32(1)
 	for {
 		resp = nil // reset
-		log.Writef(log.EventRetryPolicy, "=====> Try=%d", try)
+		// unfortunately we don't have access to the custom allow-list of query params, so we'll redact everything but the default allowed QPs
+		log.Writef(log.EventRetryPolicy, "=====> Try=%d for %s %s", try, req.Raw().Method, getSanitizedURL(*req.Raw().URL, getAllowedQueryParams(nil)))
```

Comment:
Should we be passing the user's provided "allowed" query params? I note you passed 'nil' - do we not have them at this point?

Reply:
Correct, we don't have them at this point, and it would require a new retry policy constructor to obtain them. I'd prefer to avoid that unless we get customer feedback stating that they're required.